util/src/main/java/tc/oc/pgm/util/material/Materials.java

```diff
@@ -68,48 +68,66 @@ static Material parse(String... names) {
     return BukkitUtils.parse(Material::valueOf, names);
   }
 
-  Set<Material> WEAPONS =
-      ImmutableSet.of(
-          WOODEN_SWORD,
-          STONE_SWORD,
-          GOLDEN_SWORD,
-          IRON_SWORD,
-          DIAMOND_SWORD,
-          WOOD_AXE,
-          STONE_AXE,
-          GOLD_AXE,
-          IRON_AXE,
-          DIAMOND_AXE,
-          WOOD_PICKAXE,
-          STONE_PICKAXE,
-          GOLD_PICKAXE,
-          IRON_PICKAXE,
-          DIAMOND_PICKAXE,
-          WOOD_SHOVEL,
-          STONE_SHOVEL,
-          GOLD_SHOVEL,
-          IRON_SHOVEL,
-          DIAMOND_SHOVEL,
-          WOOD_HOE,
-          STONE_HOE,
-          GOLD_HOE,
-          IRON_HOE,
-          DIAMOND_HOE,
-          BOW,
-          FLINT_AND_STEEL,
-          SHEARS,
-          STICK);
+  Set<Material> WEAPONS = ImmutableSet.of(
```

Comment:
This will probably need to go platform specific. For example tridents, or netherite tools. Fine with a TODO here given this should only effect Death messages

Reply:
I think quite a few of the methods in materials could go platform-specific, but i'm keeping them as is for the time being, future improvements i think can go in a future PR for the sake of merging this colossal one already

Reply:
Made it not platform specific but still should add the newer version items in